source/patterns/@aws-solutions-constructs/aws-lambda-elasticachememcached/README.md

```diff
@@ -0,0 +1,92 @@
+# aws-lambda-elasticachememcached module
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Stable](https://img.shields.io/badge/cfn--resources-stable-success.svg?style=for-the-badge)
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_lambda_elasticachememcached`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-lambda-elasticachememcached`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.lambdaelasticachememcached`|
+
+This AWS Solutions Construct implements an AWS Lambda function connected to an Amazon Elasticache Memcached cluster.
+
+Here is a minimal deployable pattern definition :
+
+Typescript
+``` typescript
+import { Construct } from 'constructs';
+import { Stack, StackProps } from 'aws-cdk-lib';
+import { LambdaToElasticachememcached } from '@aws-solutions-constructs/aws-lambda-elasticachememcached';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+
+new LambdaToElasticachememcached(this, 'LambdaToElasticachememcachedPattern', {
+    lambdaFunctionProps: {
+        runtime: lambda.Runtime.NODEJS_14_X,
+        handler: 'index.handler',
+        code: lambda.Code.fromAsset(`lambda`)
+    }
+});
+```
+
+Python
+```python
+
+```
+
+Java
+``` java
+
+```
```

Comment:
Missing Python and Java minimal deployments

Reply:
Yep, put those off for later :-)